components/grid/export/pdf.md

```diff
@@ -10,17 +10,223 @@ position: 1
 
 # PDF Export
 
-The [Telerik Document Processing tools](slug://dpl-in-blazor) that come with your Blazor license let you generate a PDF file based on the data in the grid.
+You can export the Grid to PDF with a click of a button. The current filter, sort, page, grouping, column order and column size are applied to the exported PDF document.
 
-The following sample projects show two ways to implement a PDF export
+When you click the Export button, your browser will receive the resulting file.
+
+#### In This Article
+
+  - [Basics](#basics)
```

Comment:
You don't really need the indentation here.

Reply:
This follows the model from the other articles. We include such indentation in every "In This Article" section - e.g. 
https://www.telerik.com/blazor-ui/documentation/components/stepper/steps/indicators